mxcube3/ui/components/SampleView/ContextMenu.js

```diff
@@ -246,8 +244,9 @@ export default class ContextMenu extends React.Component {
   savePoint() {
     if (this.props.clickCentring) {
       this.props.sampleActions.stopClickCentring();
-      this.props.sampleActions.sendAcceptCentring();
     }
+
+    this.props.sampleActions.sendAcceptCentring();
```

Comment:
I do not understand this change

Reply:
Its to accept a temporary point provided by the auto-loop centering